_posts/photography/2015-03-10-my_gear.markdown

```diff
@@ -0,0 +1,136 @@
+path: posts
+---
+
+The second most popular question following the "Why you shoot film?" is **"What gear you own and use?"**
+
+So this is the list of all the gear I own and use!
+
+## **Bronica GS-1**
+with **Zenzanon PG 100mm f/3.5** and **Zenzanon PG 65mm f/4**  
+
+this is my main camera, the one that I carry with me when I go out to photograph, from landscapes to portraits. 
+
+I am very happy with the camera as it produces very good results,
+it has good metering with the AE prism and sharp lenses.
+
+**Tip:**
+*Focusing improved dramatically when I changed the regular focusing screen with a split-screen. I recommend this upgrade to anyone with a GS-1.*
+
+The built quality is also very good, solid with good ergonomics witch makes the use of the camera pretty straightforward.
+
+The Zenzanon PG series of lenses are very good, sharp across the field, even wide open and can be found in very reasonable prices. 
+
+I use mostly the *PG 65mm* due to the style of my work and the *100mm* when I want to shoot portraits. 
+
+The only weakness of the lenses are that are a bit slow (3.5) and they don't have "character" in their bokeh.
+
+Many say that it GS1 is light and compact for a 6x7 SLR but I don't want to imagine how other 6x7 SLRs are, because for me this one is **monstrous** 
+
+I use mine with AE prism and the speedgrip which clearly adds extra weight and size but improves dramatically the handling and usage of the camera.
+
+Conclusion: 
+Bronica GS-1 is **the best 6x7 SLR** you can get in this price range (600$ with 2 lenses in 2015)
+You get **amazing results** if you know how to use it, **large negatives**, **awesome lenses**, **many accessories** and the ability to select between **different formats (6X6, 645)**.
+
+## **Contax G1** 
+with **Contax G Zeiss 28mm f/2.8 Biogon**
+
+This is a new arrival to fill the gap of a small quality 35mm camera that I will carry with me as a backup to the medium format. 
+
+I used to have an Olumpus XA2 for this job but the constraints of this camera led me to search for a better one. 
+
+I found the Contax G1 in a bargain and bought it, I don't need to say much about the Contax G system, it is one of the best 35mm cameras you can get. 
+[Ken Rockwell Review](http://www.kenrockwell.com/contax/g-system.htm)
+
+The only problem is that eventually it is not going to be a replacement of the Olympus XA2. Its bigger and more fragile, I carry the XA2 everywhere inside my everyday backpack, something you can't do with the precious Contax. 
+
+Where I use the Contax: 
+
+- Backup to the Bronica GS-1 in my photography roadtrips
+- Main camera in my urban wanderings
+- Loaded with black and white film for my black and white projects :) 
+
+## **Olympus OM2n** 
+with **Zuiko Auto-W 28mm f/3.5**
+
+This is a very good camera but I have never used it. Bought it the same time I bought the Olympus XA2 and maybe thats a good excuse for not using it. 
```

Comment:
This is a very good camera but I have never used it. Bought it around the same time I bought the Olympus XA2 and maybe that's a good excuse for not using it.
Εγώ θα ελεγα: Bought it around the same time I bought the Olympus XA2 but I ended up using XA2 almost always.